app/src/main/java/fr/free/nrw/commons/upload/FileUtils.java

```diff
@@ -59,7 +59,7 @@ public static String getPath(Context context, Uri uri) {
 
                 final String id = DocumentsContract.getDocumentId(uri);
                 final Uri contentUri = ContentUris.withAppendedId(
-                        Uri.parse("content://downloads/public_downloads"), Long.valueOf(id));
+                        Uri.parse("content://downloads/document"), Long.valueOf(id));
```

Comment:
Can you explain why this change is relevant to sharing the app from the _About_ screen?

If it's related to fixing a different issue, I'd suggest rolling it into its own PR and not including it here.

Reply:
Yes it is related to a different issue. I would close the PR immediately. Sorry for the inconvenience.